frontend/app/components/account-form.js

```diff
@@ -43,6 +46,19 @@ export default class AccountForm extends BaseFormComponent {
     });
   }
 
+  presetTeamAndFolder() {
+    let selectedTeam = this.navService.selectedTeam;
+    let selectedFolder = this.navService.selectedFolder;
+
+    if ((selectedTeam !== undefined) && this.isNewRecord){
```

Comment:
Do we need this undefined check? I mean if theres no team selected on the navbar, there is none assigned here. So even if it is undefined, we can just assign it anyways

Reply:
Yes, we don't need it.